models/engine/db_storage.py

```diff
@@ -51,6 +51,20 @@ def all(self, cls=None):
                     new_dict[key] = obj
         return (new_dict)
 
+    def get(self, cls, id):
+        """ this method is to retrieve one object """
+        for value in self.all(cls).values():
+            if value.get('id') == id:
```

Comment:
Hi dear. Please change the condition, in the part of "value.get('id')" by "value.id'". Since Get is a method from the Flask library

Reply:
Hello, thanks for your observation, I considerated and maked the change